catalog/RUB/RUB_RUBCLIM_LCZ_global_lcz_map_v1.jsonnet

```diff
@@ -0,0 +1,162 @@
+      href: 'https://doi.org/10.5194/essd-14-3835-2022',
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.5281/zenodo.6364594',
+    },
+  ],
+  keywords: [
+    'Local Climate Zones',
+    'WUDAPT',
+    'urban',
+    'landcover',
+    'climate',
+    'rub',
+  ],
+  providers: [
+    ee.producer_provider('Bochum Urban Climate Lab', 'http://www.climate.ruhr-uni-bochum.de/news/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2018-01-01T00:00:00Z', '2018-12-31T23:59:59Z'),
+  summaries: {
+    gsd: [
+      100.0,
+    ],
+    'eo:bands': [
+      {
+        name: 'LCZ',
+        description: 'The raw, pixel-based global LCZ map, with LCZ classes indicated by numbers 1-17',
+        'gee:units': '-',
+      },
+      {
+        name: 'LCZ_Filter',
+        description: 'The recommended global LCZ map, with LCZ classes indicated by numbers 1-17.
+        LCZ labels are obtained after applying the morphological Gaussian filter described in
+        Demuzere et al. (2020)',
+        'gee:units': '-',
+      },
+      {
+        name: 'LCZ_Probability',
+        description: 'A probability layer (%) that identifies how often the modal LCZ was
+        chosen per pixel (e.g. a probability of 60% means that the modal LCZ class was
+        mapped 30 times out of 50 LCZ models). This is a pixel-based probability,
+        derived from the LCZ layer',
+        'gee:units': '%',
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'LCZ',
+        lookat: {
+          lat: 40.72,
+          lon: -73.99,
+          zoom: 10,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              1,
+            ],
+            max: [
+              17,
+            ],
+            palette: [
+              '#8c0000',
+              '#d10000',
+              '#ff0000',
+              '#bf4d00',
+              '#ff6600',
+              '#ff9955',
+              '#faee05',
+              '#bcbcbc',
+              '#ffccaa',
+              '#555555',
+              '#006a00',
+              '#00aa00',
+              '#648525',
+              '#b9db79',
+              '#000000',
+              '#fbf7ae',
+              '#6a6aff'
+            ],
+            bands: [
+              'LCZ_Filter',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  'sci:doi': 'https://doi.org/10.5194/essd-14-3835-2022',
+  'gee:extra_dois': [
+    '10.5281/zenodo.6364594',
+  ],
+  'sci:citation': |||
+    Demuzere M.; Kittner J.; Martilli A.; Mills, G.; Moede, C.; Stewart, I.D.; van Vliet, J.; Bechtel, B.
+    A global map of local climate zones to support earth system modelling and urban-scale environmental science.
+    Earth System Science Data 2022, 14 Volume 8: 3835-3873.
+    [doi:10.5194/essd-14-3835-2022](https://doi.org/10.5194/essd-14-3835-2022)
+  |||,
+  'gee:terms_of_use': 'CC-BY-4.0',
+  'gee:user_uploaded': true,
```

Comment:
please also add 
'gee:skip_indexing': true

Two things that github PRs don't support yet are JS examples and image samples. Please prepare a 25x256 image that you'd like to use in the catalog and that doesn't show the basemap. Also please send us a JS example if there's something specific you'd like to highlight.